[DOCS] Updates saved objects & other docs (#139328)
elastic/kibana: docs/management/managing-tags.asciidoc

[[managing-tags]]
== Tags

> Use tags to categorize your saved objects.

Reviewer: ```suggestion
Use tags to categorize your saved objects,
```

> You can then filter for related objects based on shared tags.

Reviewer: ```suggestion
then filter for related objects based on shared tags.
```

[[settings-assign-tag]]
=== Assign a tag to an object

To assign and remove tags, you must have `write` permission
on the objects to which you assign the tags.

. Find the tag you want to assign.
. Click the actions icon
image:images/actions_icon.png[Actions icon],
and then select *Manage assignments*.

. Select the objects to which you want to assign or remove tags.
+
[role="screenshot"]
image::images/tags/manage-assignments-flyout.png[Assign flyout, width=75%]

. Click *Save tag assignments*.

[float]
[[settings-delete-tag]]
=== Delete a tag

When you delete a tag, you remove it from all saved objects that use it.

. Click the actions icon
image:images/actions_icon.png[Actions icon], and then select *Delete*.

. Click *Delete tag*.

> TIP: To assign, delete, or clear multiple tags at once,

Reviewer: ```suggestion
TIP: To assign, delete, or clear multiple tags,
```

> select their checkboxes in the *Tags* view, and then select

Reviewer: ```suggestion
select them in the *Tags* view, and then select
```

> the desired action from the *selected tags* menu.

Reviewer: ```suggestion
the action from the *selected tags* menu.
```
